devel/libuv/Portfile

```
@@ -21,14 +21,13 @@ subport libuv-devel {
     legacysupport.newest_darwin_requires_legacy 15
 
     github.setup \
-                libuv libuv 1.47.0 v
+                libuv libuv 1.51.0 v
     # Change github.tarball_from to 'releases' or 'archive' next update
     github.tarball_from tarball
-    revision    0
-
-    checksums   rmd160  6510cd246bb2062874a5cb37e92a1c727a7d2655 \
-                sha256  72a187104662b47f2a2b204da39d2acb05cf22a4fcb13ceaebe3b0ed0c0e2e43 \
-                size    1654769
+    revision    1
+    checksums   rmd160  40cf693021f7584ff4aeffcbf2020d0e00139005 \
+                sha256  2ceca1a7577633cf92794db5bf5512370f6cd45a5746d6e14f8c20aeab0a547b \
+                size   1691252 
```

This doesn't look like it's aligned correctly, and it has trailing whitespace.
```suggestion
                size    1691252
```
However, please follow the instructions in the comment and change `github.tarball_from` from `tarball` to either `releases` or `archive`, whichever is correct for this software; then the checksums will change again.

Since you are increasing the `version`, the `revision` should be `0`.